nextcord/member.py

```diff
@@ -278,6 +283,7 @@ def __init__(self, *, data: MemberWithUserPayload, guild: Guild, state: Connecti
         self._timeout: Optional[datetime.datetime] = utils.parse_time(
             data.get("communication_disabled_until")
         )
+        self.flags = MemberFlags._from_value(data["flags"])
```

Comment:
```suggestion
        self.flags: MemberFlags = MemberFlags._from_value(data["flags"])
```
explicit typing, mostly for dumber type checkers but also for consistency.